.github/workflows/nightly-sigar.yml

```diff
@@ -0,0 +1,24 @@
+name: Akka Nightly Sigar Build
```

Comment:
I'm confused when I see Sigar and Cluster Metrics, I think we can name everything "Cluster Metrics", wdyt?

Reply:
I guess the difference is that this job runs the cluster metrics tests with 'sigar' enabled, and the regular build runs them without that library?

Reply:
That is my guess too.. let me rename.